blocks/init/src/Blocks/components/accordion/accordion.php

```diff
@@ -24,6 +24,7 @@
 $accordionTitle = Components::checkAttr('accordionTitle', $attributes, $manifest);
 $accordionContent = Components::checkAttr('accordionContent', $attributes, $manifest);
```

Comment:
You have a variable name mismatch with the accordion block here, as it gets an `accordionAccordionContent` attribute. This results in accordion having no content output on frontend.

```php
echo Components::render( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
	'accordion',
	Components::props('accordion', $attributes, [
		'accordionAccordionContent' => $innerBlockContent
	])
);
```